packages/core/crates/core-strategy/src/promiscuous.rs

```diff
@@ -250,41 +249,41 @@ mod tests {
 
         assert_eq!(strat.name(), "promiscuous");
 
-        let alice = PeerId::random();
-        let bob = PeerId::random();
-        let charlie = PeerId::random();
-        let eugene = PeerId::random();
-        let gustave = PeerId::random();
+        let alice = Address::from_str("0x5cb1d93aea1fc219a936a708576bf553042993ea").unwrap();
+        let bob = Address::from_str("0xcc57a920e27f6eaa78c3ccb7976c0fb1e4b94ea1").unwrap();
+        let charlie = Address::from_str("0xf6a2be4680ab2051f2906922c210da0508553ce1").unwrap();
+        let eugene = Address::from_str("0x242c983f08f896e1b3a51ec6d646c20bf1b494fc").unwrap();
+        let gustave = Address::from_str("0xf6c287af75350dc241255f369368044a02c65160").unwrap();
 
         let peers = HashMap::from([
             (alice.clone(), 0.1),
             (bob.clone(), 0.7),
             (charlie.clone(), 0.9),
-            (PeerId::random(), 0.1),
+            (Address::random(), 0.1),
```

Comment:
Would be good to add `Address::random()` function

Reply:
Just did that